lwk/lwkwallet_test.go

```diff
@@ -0,0 +1,32 @@
+package lwk_test
+
+import (
+	"testing"
+
+	"github.com/elementsproject/peerswap/lwk"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSatPerKVByteFromFeeBTCPerKb(t *testing.T) {
+	t.Parallel()
+	t.Run("below minimum minimumSatPerByte", func(t *testing.T) {
+		t.Parallel()
+		var (
+			txsize      int64 = 1000
+			FeeBTCPerKb       = 0.0000001
+		)
+		got := lwk.SatPerKVByteFromFeeBTCPerKb(FeeBTCPerKb).GetFee(txsize)
+		want := lwk.Satoshi(100)
+		assert.Equal(t, want, got)
+	})
+	t.Run("above minimum minimumSatPerByte", func(t *testing.T) {
+		t.Parallel()
+		var (
+			txsize      int64 = 1000
+			FeeBTCPerKb       = 0.000002
```

Comment:
The test above has a fee in btc/kvB of 10^-7 while this test has a fee of 10^-6 and both have a tx size of 10^3 and want a result in 10^2. This does not seem right.

Reply:
In the test case above, the `SatPerVByte` is 10^-2 and the `minimumFee` of 10^-1 is used. 
In the lower case, the `SatPerVByte` would be `SatPerVByte`, so the 10^-1 digits would be the same. 

The name was confusing and has been corrected.

[`7045055` (#288)](https://github.com/ElementsProject/peerswap/pull/288/commits/7045055a00ffb06466bf9312fc23cb1a41e3e364)